programs/protocol_event/src/context.rs

```diff
@@ -117,3 +117,11 @@ pub struct CreateParticipant<'info> {
     #[account(address = system_program::ID)]
     pub system_program: Program<'info, System>,
 }
+
+#[derive(Accounts)]
+pub struct UpdateParticipant<'info> {
+    #[account(mut, has_one = authority)]
+    pub participant: Account<'info, Participant>,
+    #[account(mut)]
```

Comment:
Don't think it needs to be `mut`, does it?

Reply:
ah right you are, only need `mut` when creating accounts